syncstorage/storage/sql/queries_mysql.py

```diff
@@ -10,13 +10,23 @@
 
 # MySQL's non-standard DELETE ORDER BY LIMIT is incredibly useful here.
 
-PURGE_SOME_EXPIRED_ITEMS = "DELETE FROM %(bso)s "\
-                           "WHERE ttl < (UNIX_TIMESTAMP() - :grace) " \
-                           "ORDER BY ttl LIMIT :maxitems"
+PURGE_SOME_EXPIRED_ITEMS = """
+    DELETE FROM %(bso)s
+    WHERE ttl < (UNIX_TIMESTAMP() - :grace)
+    ORDER BY ttl LIMIT :maxitems
+"""
+
+PURGE_BATCHES = """
+    DELETE FROM batch_uploads
+    WHERE batch < (UNIX_TIMESTAMP() - :lifetime - :grace) * 1000
```

Comment:
Note that we now pass in the batch lifetime as a separate parameter, rather than hard-coding it into the calculation of the grace period.  This has further simplified the tests for this code.